### PR TITLE
Running the project

### PR DESCRIPTION
**Pull request by kalkedan**

### CODE REVIEW

**Comment by n-grounds** [comment]
Should this be merging into `dev` instead of `main`?

**Comment by kalkedan** [comment]
yes it should be merging to the dev branch

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from'react'
+const Home =() => {
```
Comment (n-grounds) [inline src/components/Home.js:2]: curious about why you use this syntax here and `function About() {` in the About.js file above.  As I understand it, they are both achieving the same thing.

**Comment by kalkedan** [comment]
I was just learning two ways of doing it on the tutorials I was watching that is why. Sent from Mail for Windows 10 From: n-groundsSent: Thursday, June 10, 2021 7:10 PMTo: logan-giese/whiteboard-course-managementCc: kalkedan; AuthorSubject: Re: [logan-giese/whiteboard-course-management] Running the project ***@***.*** commented on this pull request.In src/components/Home.js:> @@ -0,0 +1,8 @@+import React from'react'+const Home =() => {curious about why you use this syntax here and function About() { in the About.js file above. As I understand it, they are both achieving the same thing.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by n-grounds** [comment]
> yes it should be merging to the dev branch

I think you can edit the Pull Request to update what branch you are requesting to merge into, in that case.

**Review by logan-giese — APPROVED** [review]
Tested and it works

**Pull request merged**